src/vivarium_public_health/interactive.py

```diff
@@ -0,0 +1,54 @@
+import vivarium.interface.interactive as vi
+from vivarium.framework.configuration import build_model_specification
+from vivarium.framework.plugins import PluginManager
+from vivarium.config_tree import ConfigTree
+
+from typing import Mapping, Union, List
+
+ARTIFACT_PLUGIN = {
+    'optional': {
+        'data': {
+            'controller': 'vivarium_public_health.dataset_manager.ArtifactManager',
+            'builder_interface': 'vivarium_public_health.dataset_manager.ArtifactManagerInterface'
+        }
+    }
+}
+
+
+def initialize_simulation(components: List, input_config: Union[Mapping, ConfigTree]=None,
+                          plugin_config: Mapping=None) -> vi.InteractiveContext:
+    plugin_config = ARTIFACT_PLUGIN.update(plugin_config) if plugin_config else ARTIFACT_PLUGIN
+    return vi.initialize_simulation(components, input_config, plugin_config)
+
+
+def setup_simulation(components: List, input_config: Union[Mapping, ConfigTree]=None,
+                     plugin_config: Mapping=None) -> vi.InteractiveContext:
+    simulation = initialize_simulation(components, input_config, plugin_config)
+    simulation.setup()
+
+    return simulation
+
+
+def initialize_simulation_from_model_specification(model_specification_file: str) -> vi.InteractiveContext:
```

Comment:
Here as well.  Drop line 40 and replace the explicit constructions with a call to the locally defined `initialize_simulation`